Gem/Code/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ ly_add_target(
             Gem::Atom_Feature_Common.Static
             Gem::Atom_Component_DebugCamera.Static
             Gem::AtomSampleViewer.Lib.Static
+            Gem::Profiler.Static
```

Comment:
Aren't we eventually going to move to a way to set profilers dynamically?

Reply:
This is a special case for Atom Sample Viewer to access the profiler visualization because it does things differently.  The answer to your question though is yes.